DependencyInjection/Configuration.php

```diff
@@ -54,10 +54,10 @@ public function getConfigTreeBuilder()
                 ->booleanNode('use_flash_notifications')->defaultTrue()->end()
                 ->booleanNode('use_username_form_type')->defaultTrue()->end()
                 ->arrayNode('from_email')
-                    ->addDefaultsIfNotSet()
+                    ->isRequired()
```

Comment:
should not be required, as it is possible to use FOSUserBundle without sending emails (depending on the features you use)